assets/sass/components/header.scss

```diff
@@ -1,17 +1,72 @@
 .header {
+  position: relative;
   align-items: center;
   display: flex;
   flex-wrap: wrap;
-  margin-bottom: 1.2em;
-  padding: 0.6em 2%;
+  margin-bottom: 4em;
+  padding: 0.6em 2% 0.6em 2%;
+  border-bottom: 8px solid $color-primary;
```

Comment:
Try not to mix units too much. `margin` and `padding` are `em`s, let’s make those non-`px` also.